projects/kimageformats/Dockerfile

```diff
@@ -16,6 +16,9 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder
 RUN apt-get install --yes cmake
+RUN apt-get install --yes autoconf
+RUN apt-get install --yes libtool
+RUN apt-get install --yes pkg-config
```

Comment:
You should do update every time you do an install, otherwise caching can make your build fail randomly (your update will be out of date)

Am not sure about the exact details, but noticed this elsewhere: https://github.com/google/oss-fuzz/pull/8540#discussion_r975389900

Reply:
`RUN apt-get update && apt-get install -y cmake autoconf libtool pkg-config` should be good

Reply:
See https://stackoverflow.com/questions/52067577/apt-get-update-in-a-dockerfile-destroys-app-consistency